content/package.json

```diff
@@ -4,7 +4,7 @@
   "main": "index.js",
   "license": "MPL-2.0",
   "scripts": {
-    "import": "node scripts/import-from-mysql.js"
+    "import": "node index.js"
```

Comment:
I actually don't use these. I use 
```bash
node content import ...
```
or 
```bash
node content build
```
I speculate that we might tidy up some of these things later in the root `/package.json` 's script.

Reply:
Yeah, it doesn't really matter. I didn't use it either, but what was there was clearly incorrect so I thought I'd adjust it to something that at least existed.